Resources/public/templates/testonearticlesview.hbt

```diff
@@ -0,0 +1,11 @@
+{{#if previous}}
+<a href="#" data-location-id="{{previous.locationId}}" class="article__link article__link--previous">Back to {{previous.name}}</a>
+{{/if}}
+<header>
```

Comment:
Jeśli mamy tylko jeden nagłówek bez żadnych dodatkowych rzeczy to tag `<header/>` jest tutaj zbędny.